spec/services/early_career_teachers/create_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+RSpec.describe EarlyCareerTeachers::Create do
+  let(:user) { create :user }
+  let(:school) { create :school }
+  let(:pupil_premium_school) { create :school, :pupil_premium_uplift }
+  let(:sparsity_school) { create :school, :sparsity_uplift }
+  let(:uplift_school) { create :school, :sparsity_uplift, :pupil_premium_uplift }
+  let(:cohort) { create :cohort, :current }
+  let(:mentor_profile) { create :mentor_profile }
+
+  it "creates an Early Career Teacher Profile record" do
+    before_count = EarlyCareerTeacherProfile.count
+    early_career_teacher_profile = described_class.call(email: user.email, full_name: user.full_name, school_id: school.id, cohort_id: cohort.id, mentor_profile_id: mentor_profile.id)
+    after_count = EarlyCareerTeacherProfile.count
+    expect(after_count).to eq(before_count + 1)
```

Comment:
I would make this its own test, without checking anything else. And use the form of 
```
    expect {
      described_class.call(email: user.email, full_name: user.full_name, school_id: school.id, cohort_id: cohort.id, mentor_profile_id: mentor_profile.id)
    }.to change { EarlyCareerTeacherProfile.count }.by(1)
```

Also, should we add a test for the edge case when user does not exist? And the test could say that 'when the user does not exist, it creates a new user'.